src/kernels/MIOpenLRNFwd.cl

```diff
@@ -410,10 +410,12 @@ MIOpenLRNAcrossChannels4(const __global _FLOAT* bottom,
     int b               = get_global_id(2); // batch
     MLO_READ_TYPE accum = 0;
     MLO_READ_TYPE bot_in2[MLO_LRN_KERNEL_SZ];
+    MLO_READ_TYPE bot_in[MLO_LRN_KERNEL_SZ];
     int c_i = 0, c_o = 0;
     for(int i = 0; i < MLO_LRN_KERNEL_SZ; ++i)
     {
         bot_in2[i] = 0;
+        bot_in[i] = 0;
```

Comment:
Hi @densamoilov, first of all, thank you very much for raising this PR!
We have some issues with Clang Format Tidy with this line and several lines below, likely due to a tab in front of "=" sign.
The best way to get rid of the format issue is to follow procedure outlined here:
https://github.com/ROCmSoftwarePlatform/MIOpen/wiki/How-to-format-code 

Thanks!

Reply:
You can use something like this to only format files changed from baseline if you want (this implies WSL, but is easily modified to be called from native linux if you want):
```Powershell
Param(
	[Parameter(ParameterSetName="Work")]
	[String]$Distro = "Ubuntu-20.04",
	[Parameter(ParameterSetName="Work")]
	[String]$ClangFormat = "clang-format-10",
	[Parameter(ParameterSetName="Work")]
	[Switch]$Log,
	[Parameter(ParameterSetName="Work")]
	[Switch]$SingleThread,
	[Parameter(ParameterSetName="List", Mandatory=$true)]
	[Switch]$List,
	[Switch]$All)

Import-Module PoshRSJob

$branch = git rev-parse --abbrev-ref HEAD
$files = @()

if ($All)
{
	$files += git ls-tree --full-tree -r --name-only HEAD
}
else
{
	$files += git diff --name-only remotes/origin/develop..$branch
	$files += git clean -dn | % { $_.Substring(13) }
}

$files = $files| ? { $_ -Match "\.h$|\.hpp$|\.c$|\.cpp$|\.h\.in$|\.hpp\.in$|\.cpp\.in$|\?CMakeLists.txt" }

if ($List)
{
	$files
	return
}

$repoRoot = git rev-parse --show-toplevel
$jobs = @()

$Worker = {
	Param($file, $repoRoot, $Log, $Distro, $ClangFormat)
	Set-Location $repoRoot
	$out = wsl --distribution "$Distro" --exec "$ClangFormat" -i -style=file "$file"
	if ($Log) { $out }
	if ($LASTEXITCODE -eq 0) { if ($Log) { "Finished formating `"$file`"." } }
	else { "Error formating `"$file`"." }
}

$jobId = 0

foreach ($file in $files)
{
	$jobId++

	if (-not (Test-Path -PathType Leaf $file))
	{
		if ($Log) { Write-Host -ForegroundColor Cyan "File doesn't exist: `"$file`"`r" }
		Write-Progress -Id 0 -Activity "clang-format" -CurrentOperation "Starting" -PercentComplete (100 * $jobId / $files.Count) -Status "File doesn't exist: `"$file`""
	}
	else
	{
		if ($Log) { Write-Host -ForegroundColor Cyan "wsl --distribution $Distro --exec $ClangFormat -i -style=file $file" }
		Write-Progress -Id 0 -Activity "clang-format" -CurrentOperation "Starting" -PercentComplete (100 * $jobId / $files.Count) -Status "clang-format-3.8 -i -style=file `"$file`""
		if ($SingleThread) { & $Worker $file $repoRoot $Log $Distro $ClangFormat }
		else { $jobs += Start-RSJob -ScriptBlock $Worker -ArgumentList $file, $repoRoot, $Log, $Distro, $ClangFormat }
	}
}

Write-Progress -Id 0 -Activity "clang-format" -Completed

Wait-RSJob -Job $jobs -ShowProgress | Out-Null
$jobs | Receive-RSJob;
$jobs | Remove-RSJob;
```